### ISSUE TITLE
Privacy policy

### ISSUE THREAD

**Issue by fabianski7** [lrstanley/links#17]
This redirector seems to be quite popular (almost 3 million links generated) so I think there should be some link on the page with the privacy policy, or right here at github.

**Comment by lrstanley** [comment]
Will leave this open, so when I have some free time, I can probably write something up.

Note that a good portion (about 50% from my recent calculation) that are shortened are spam bots using it to shorten phishing links. Have to work on mitigation solutions, and Google SafeBrowsing (https://github.com/lrstanley/links/issues/16), which will hopefully cut down on the number of these potentially malicious expanded links.

**Comment by lrstanley** [comment]
The TL:DR though, is that (as of today) for links.wtf:

1. The only data that is stored in the DB is the same data you see in this repository. None of the intermediary web servers/load balancers/etc store data for longer than 30d (ip, route/path,  GET parameters, useragent).
2. None of the data is currently moved outside of the server in which it is hosted on.
3. None of the data for links.wtf is shared with anyone other than I.
4. Requests for removal of data isn't easily possible today due to lack of account information to map shortened links to. Will only remove data if requested by hosting provider, legal request, or links which I deem potentially dangerous to end users, at this time.

Have been thinking about API key system/oauth system to map back to identities, to allow managing their own links/delete their data, but not implemented today.

**Comment by lrstanley** [comment]
Hasn't ran the actual links.wtf site in some time, mostly due to not having the time to keep up with resolving phishing reports (due to bots getting through spam protection), with my current hosting provider (@ OVH, all reports are automated unless you're an enterprise, so they essentially won't listen when I tell them the links.wtf site wasn't hosting anything, it's simply a redirection service). Was really annoying, because I also had Cloudflare in front of the service, which didn't stop the bots, and Cloudflare still forward those types of reports through to the hosting provider.

If I decide to ever run it again, I will revisit.
